editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/restconfiguration/RestConfigEditor.java

```diff
@@ -325,33 +299,46 @@ private void getImages() {
 						.getEntry(RestConfigConstants.IMG_DESC_DELETE)));	
 	}
 
-	private ToolBar createToolbar(Composite parent) {
+	private ToolBar createRestConfigurationToolbar(Composite parent) {
 		ToolBarManager toolBarManager = new ToolBarManager(SWT.FLAT);
 		ToolBar toolbar = toolBarManager.createControl(parent);
 		toolbar.setBackground(Display.getCurrent().getSystemColor(SWT.COLOR_WHITE));
 
-		// Add action to the tool bar
-		AddAction action = new AddAction();
-		toolBarManager.add(action);
-		action.setEnabled(false);
-		DeleteAction daction = new DeleteAction();
-		toolBarManager.add(daction);
-		daction.setEnabled(false);
+		addRestConfigAction = new AddRestConfigurationAction(this, mImageRegistry);
+		toolBarManager.add(addRestConfigAction);
+
+		deleteRestConfigAction = new DeleteRestConfigurationAction(this, mImageRegistry);
+		toolBarManager.add(deleteRestConfigAction);
 
 		toolBarManager.update(true);
 		return toolbar;
 	}
-	
-	private ToolBar createRestConfigurationToolbar(Composite parent) {
+
+	private ToolBar createRestTabToolbar(Composite parent) {
 		ToolBarManager toolBarManager = new ToolBarManager(SWT.FLAT);
 		ToolBar toolbar = toolBarManager.createControl(parent);
 		toolbar.setBackground(Display.getCurrent().getSystemColor(SWT.COLOR_WHITE));
 
-		addRestConfigAction = new AddRestConfigurationAction(this, mImageRegistry);
-		toolBarManager.add(addRestConfigAction);
+		addRestElementAction = new AddRestElementAction(this, mImageRegistry);
+		toolBarManager.add(addRestElementAction);
 
-		deleteRestConfigAction = new DeleteRestConfigurationAction(this, mImageRegistry);
-		toolBarManager.add(deleteRestConfigAction);
+		deleteRestElementAction = new DeleteRestElementAction(this, mImageRegistry);
+		toolBarManager.add(deleteRestElementAction);
+
+		toolBarManager.update(true);
+		return toolbar;
+	}
+
+	private ToolBar createRestOperationTabToolbar(Composite parent) {
+		ToolBarManager toolBarManager = new ToolBarManager(SWT.FLAT);
+		ToolBar toolbar = toolBarManager.createControl(parent);
+		toolbar.setBackground(Display.getCurrent().getSystemColor(SWT.COLOR_WHITE));
```

Comment:
need to use FormToolkit to create/adapt toolbar to have it coherent with possible different Eclipse theme